### PR TITLE
#69133 prevent images from jumping on load

### PR DESCRIPTION
**Pull request by kamil-jagoda**

### CODE REVIEW

**Review by anqaka — APPROVED** [review]
Great work! I really like it!

File: Magento_Catalog/styles/_module.scss [hunk 1]
```diff
@@ -1 +1,2 @@
 @import 'modules/fotorama';
+@import 'modules/image-ratio';
```
Comment (pawel-snowdog) [inline Magento_Catalog/styles/_module.scss:2]: I'm not sure that this is the best place to import these styles... I saw that this class is used in `/Magento_Theme/templates/html/picture.phtml` so it has global usage. Maybe some mixin in Atoms would be better?

File: Magento_Theme/templates/html/picture.phtml [hunk 2]
```diff
@@ -113,10 +129,49 @@
...
+                lazyload
+                <?= $block->escapeHtmlAttr($imgClass) ?>
+            "
             data-src="<?= $block->escapeUrl($mediaUrl . $imgDefault) ?>"
             alt="<?= $block->escapeHtmlAttr($pictureAlt) ?>"
             src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAABCAQAAABN/Pf1AAAAC0lEQVR42mNkwAIAACoAAgu1Hc4AAAAASUVORK5CYII="
         />
     </picture>
+    <!-- styles are rendered only if values for image ratio for mq are provided. Content Type (picture.phtml) id is required -->
+    <?php if($pictureAspectRatio && $isAspectratioMq && $pictureId): ?>
```
Comment (pawel-snowdog) [inline Magento_Theme/templates/html/picture.phtml:142]: space after `if`

File: etc/view.xml [hunk 3]
```diff
@@ -7,20 +7,20 @@
                 <height>140</height>
             </image>
             <image id="cart_cross_sell_products" type="thumbnail">
-                <width>320</width>
-                <height>320</height>
+                <width>224</width>
```
Comment (pawel-snowdog) [inline etc/view.xml:10]: Please verify image sizes for 767px screen width. Grid product tiles has 368px width and image is stretched and looks bad. I think we should consider using max possible width as minimum default for all thumbnails

**Review by pawel-snowdog — CHANGES_REQUESTED** [review]
Verify comments

File: Magento_Theme/templates/html/picture.phtml (hunk 2 above)
Comment (anqaka) [reply Magento_Theme/templates/html/picture.phtml:142]: 👍

File: Magento_Catalog/styles/_module.scss (hunk 1 above)
Comment (anqaka) [reply Magento_Catalog/styles/_module.scss:2]: I moved styles to components -> molecules/image, I'll implement this solution in components in related task: https://dev.snowdog.pro/issues/71300

File: etc/view.xml (hunk 3 above)
Comment (anqaka) [reply etc/view.xml:10]: good catch, fixed

**Review by pawel-snowdog — APPROVED** [review]

**Pull request merged**